Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4075,6 +4075,10 @@ - (void)markVisibleMessagesAsRead
 
     uint64_t lastVisibleTimestamp = self.lastVisibleTimestamp;
```

Comment:
Should we make lastVisibleTimestamp a `NSNumber *`?

Reply:
We could - but making this change would clutter up a few places where we (correctly) rely on the nil == 0 for concise assignment. I'm not sure it's worth it.